synfig-studio/plugins/lottie-exporter/lottie-exporter.py

```diff
@@ -62,7 +62,11 @@ def gen_html(file_name):
     Returns:
         (None)
     """
-    bodymovin_path = os.path.join(os.path.dirname(sys.argv[0]), "bodymovin.js")
+    if len(settings.blur_dictionary) != 0:
```

Comment:
Is this still needed? Why do we have test_bodymovin.js in here?

Reply:
test_bodymovin.js is required for rendering the blur as it is not supported in the version which we were using. 
I am not sure whether moving completely to the newer version will be beneficial to us as it may lead to different results now.

Reply:
alright